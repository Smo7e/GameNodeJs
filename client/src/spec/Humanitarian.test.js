import React from "react";
import { render } from "@testing-library/react";
import Humanitarian from "../component/Heroes/component/Humanitarian"; 

describe("Humanitarian Component", () => {
  it("renders correctly", () => {
    const Heroes = jest.fn(); 
    const { asFragment } = render(<Humanitarian heroes={Heroes} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
