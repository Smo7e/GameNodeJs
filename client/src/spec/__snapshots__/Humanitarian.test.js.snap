// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Humanitarian Component renders correctly 1`] = `
<DocumentFragment>
  <div>
    <button
      class="arrow-2-heroes"
    />
    <button
      class="arrow-3-heroes"
    />
    <div
      class="image-humanitarian-heroes"
    />
    <div
      class="panel-heroes"
    >
      <div
        class="text-heroes"
      >
        “Гуманитарий” - имеет обычное здоровье в 100 единиц, увеличенный вдвое урон по парам “Русский язык”, “Английский язык”. Предметы типа “Книга улучшают характеристики в 1.5 раза сильнее.
      </div>
    </div>
  </div>
</DocumentFragment>
`;
